websites/D/Dizipub/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "author": {
+      "name": "Snowy ✨",
+      "id": "693445343332794408"
+    },
+    "service": "Dizipub",
+    "description": {
+      "en": "Dizipub is a website that you can watch your favorite series and films.",
+      "tr": "Dizipub, favori film ve dizilerinizi kolayca bulabileceğiniz bir film sitesidir."
+    },
+    "url": ["dizipub.net"],

```

Comment:
url shouldnt be in an array if its only 1

Reply:
Done.